Makefile.toml

```diff
@@ -9,8 +9,8 @@ skip_core_tasks = true
 RUST_OPTIMIZER_VERSION = "0.15.0"
 # Use rust version from rust-optimizer Dockerfile (see https://github.com/CosmWasm/rust-optimizer/blob/main/Dockerfile#L1)
 # to be sure that we compile / test against the same version
-RUST_VERSION = "1.72.0"
-NIGHTLY_VERSION = "nightly-2024-02-12"
+RUST_VERSION = "1.75.0"
```

Comment:
lets run 1.79.0. for some reason 1.75 takes 30 min for me to compile the latest osmosis-std (after they upgraded go version to 1.23)

Reply:
```suggestion
RUST_VERSION = "1.79.0"
```